src/index.js

```diff
@@ -368,13 +368,21 @@ router.post('/clinvarCountsForGene', async (ctx) => {
 // genepanel endpoints
 //
 router.get('/clinphen', async (ctx) => {
-
+ 
```

Comment:
I had added a console.log statement here to test. Looks like an extra space got added in the process.